extras/man/http.1

```diff
@@ -570,9 +570,10 @@ installation (only the supported ones are shown here).
 
 
 A string in the OpenSSL cipher list format.
+tls1.3 ciphers are always present regardless of your cipher list.
```

Comment:
This may need a bit more context or outsider-friendly rewording. WDYT?

``` suggestion
Applies only for TLS v1.2 and lower. TLS v1.3 connections always use the appropriate cipher suites available for that protocol version, regardless of this setting.
```

Reply:
see https://docs.python.org/3/library/ssl.html#ssl.SSLContext.set_ciphers

we can't do anything about it. unless we're using http3 over quic.

Reply:
I'm not challenging anything from that. Most APIs are only handling tls12 ciphers, and don't bother with lower-level ctxs to set tls13 suites, esp. in OpenSSL world. — I'm only raising the need to explain that to consumers not familiar with the distinction in the new protocol RFC.

Reply:
Of course, there's certainly a lot of room to improve the wording for a wider range of users. The primary goal of this PR is to move forward on the feature level and to detach ourselves from Requests.
Follow up PRs are of course, welcomed.